provider-directory-nginx-proxy/run-local.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+#=============================================
+
+usage() {
+  cat <<EOF
+Usage:
+  ${0} [options]
+
+Options:
+  --debug            Turn on debug output
+  -h|--help          Display this menu
+  -n|--nginx-conf    Location of nginx-conf file to use
+  -s|--secrets-conf  Location of secrets file for environment substitution
+
+${1}
+EOF
+exit 1
+}
+
+#=============================================
+
+init() {
+  set -euo pipefail
+
+  WORK="$(readlink -f $(dirname $0))"
+  cd ${WORK}
+
+  # provider-directory was renamed. In the unlikely scenario that someone is using the old name, allow override of the var.
+  DEPLOYMENT_UNIT="${DEPLOYMENT_UNIT:-health-apis-provider-directory-api-deployment}"
```

Comment:
The above comment is not applicable, and this line can be simplified to `DEPLOYMENT_UNIT="health-apis-provider-directory-deployment"`